### ISSUE TITLE
Remove manage.py references in development/docker-compose.vscode-rdb.yml

### ISSUE THREAD

**Issue by gsnider2195** [nautobot/nautobot#3476]
### Proposed Changes

manage.py was removed from `next`, this docker compose file needs to be updated to remove reference to manage.py

**Comment by jathanism** [comment]
As discussed on Slack, this can be replaced by:

- Remove `if __name__ == "__main__" stanza from end of `nautobot/core/cli.py`
- Move `nautobot/core/cli.py` to `nautobot/core/cli/__init__.py`
- Drop `nautobot/core/cli/__main__.py` with the following contents:

```python
if __name__ == "__main__":
    from nautobot.core.cli import main

    main()
```

This makes it where apps that require `manage.py` or `-m manage` (such as `debugpy`) can instead use `-m nautobot.core.cli`.

**Comment by glennmatthews** [comment]
Addressed by #3731.
